client/src/components/list/List.tsx

```diff
@@ -179,12 +184,13 @@ function List(props: ListProps) {
     },
   };
 
-  function onGridReady() {
+  function onGridReady(params: any) {
     const allColumnIds: string[] = [];
     gridRef.current!.api!.getAllGridColumns()!.forEach((column) => {
       allColumnIds.push(column.getId());
     });
     gridRef.current!.api!.autoSizeColumns(allColumnIds, false);
+    params.api.sizeColumnsToFit();
```

Comment:
why are we perform multiple sizing procedure?

Reply:
one is for fitting the content inside each column. another is for fitting the columns on the container. As I know